Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/uploadtransformation/UploadTransformationManager.java

```diff
@@ -0,0 +1,31 @@
+package edu.asu.spring.quadriga.service.impl.uploadtransformation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.service.uploadtransformation.IUploadTransformationManager;
+import edu.asu.spring.quadriga.dao.impl.uploadTransformation.UploadTransformationDAO;
+import edu.asu.spring.quadriga.dao.uploadtransformation.IUploadTransformationDAO;
+import edu.asu.spring.quadriga.dto.UploadTransfomationFilesDTO;
+
+
+@Service
+public class UploadTransformationManager implements
```

Comment:
same here. Just TransformationManager should be fine. Unless it is handling just the upload.